packages/kotti-ui/source/kotti-field-select/constants.ts

```diff
@@ -31,6 +31,18 @@ export const KOTTI_FIELD_SELECT_SUPPORTS: KottiField.Supports = {
 
 export const KOTTI_FIELD_SINGLE_SELECT_PROPS = {
 	...KOTTI_FIELD_SELECT_PROPS,
+	actions: {
+		default: () => null,
+		type: Array,
+		validator: (value: unknown[]): value is KottiFieldSingleSelect.Action[] =>
+			value.every(
+				(action: any): action is KottiFieldSingleSelect.Action =>
+					typeof action === 'object' &&
+					action !== null &&
```

Comment:
you seem to depend on `action.value` in the template as your key but don't check that such an attribute lives on your `action` object here

Reply:
value doesnt exist in action. It was a mistake from me